python-spec/src/somacore/ephemeral/collections.py

```diff
@@ -51,7 +52,9 @@ def open(cls, *args, **kwargs) -> NoReturn:
         )
 
     @classmethod
-    def exists(cls, uri: str, *, context: Any = None) -> Literal[False]:
+    def exists(
+        cls, uri: str, *, context: Optional[ContextBase] = None
+    ) -> Literal[False]:
```

Comment:
In light of the above, this can be left as `context: Any = None` (i.e. no changes to this file at all).